news/_posts/2021-05-28-Flannery-defense.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: news
+title: Jessica Flannery defends dissertation
+author: Taylor Salo
+category: news
+tags: []
+image: /assets/images/team/jessica-flannery.jpg
+excerpt: Graduate student Jessica Flannery has successfully defended her dissertation
+---
+
+[Jessica Flannery](/team/flannery-jessica) successfully defended her dissertation,
+"Neurobiological Impact of HIV Infection and Chronic Cannabis Use", on May 28, 2021,
+and has now received her Ph.D. in Cognitive Neuroscience.
+
+Jessica will soon start a post-doc at UNC Chapel Hill in two labs:
+Dr. Eva Telzer's [Developmental Social Neuroscience Lab](https://dsnlab.web.unc.edu) and
+Dr. Kristen Lindquist's [Carolina Affective Science Laboratory](https://www.carolinaaffectivescience.com).
+
+Congratulations Jessica!
```

Comment:
@Flanneryg3 is there anything you'd like to see changed here to make it more customized? Otherwise it looks really similar to Katie and Ranjita's announcements.

Reply:
looks good, thanks!